art/metrics/metrics.py

```diff
@@ -26,17 +26,25 @@
 
 import numpy as np
 import numpy.linalg as la
+import matplotlib.pyplot as plt
 from scipy.optimize import fmin as scipy_optimizer
 from scipy.stats import weibull_min
 
 from art.attacks import FastGradientMethod
+from art.attacks import HopSkipJump
 from art.utils import random_sphere
 from art import NUMPY_DTYPE
 
 logger = logging.getLogger(__name__)
 
 SUPPORTED_METHODS = {
     "fgsm": {"class": FastGradientMethod, "params": {"eps_step": 0.1, "eps_max": 1., "clip_min": 0., "clip_max": 1.}},
+    "hsj":{"class": HopSkipJump, "params":{
+                  'max_iter': 50,
+                  'max_eval': 10000,
+                  'init_eval': 100,
+                  'init_size': 100,
+                  }}
```

Comment:
Could you please change this to 1 or 2 lines similar as for fgsm?

Reply:
Done